ppa/archive/views.py

```diff
@@ -320,8 +319,35 @@ def get_context_data(self, **kwargs):
         })
         return context
 
+    def last_modified(self):
+        '''override last modified logic to work with Solr'''
+
+        # override sort to return most recent modification date,
+        # only return last modified value and nothing else
+
+        # use most recent modification time of anything in the index,
+        # since any search on the archive page coud change based
+        # on work or text content changing in the index, (could add
+        # or remove results from any particular set)
```

Comment:
wish there was a smarter way to do this, but it seems really hard, if not downright impossible.

Reply:
Agreed. I know in the past I've used the most recently modified date of anything in the current result set, but that doesn't account for items that might previously have been in that result list and aren't anymore. This still may not handle deletions - although with our suppression logic in PPA it should work pretty well.  And even though it might be more generous than necessary, it's still a lot better than nothing.